spec/api/v1/australian_trade_leads_spec.rb

```diff
@@ -3,8 +3,10 @@
 describe 'Australian Trade Leads API V1', type: :request do
   before(:all) do
     TradeLead::Australia.recreate_index
-    TradeLead::AustraliaData.new(
-      "#{Rails.root}/spec/fixtures/trade_leads/australia/trade_leads.csv").import
+    VCR.use_cassette('importers/trade_leads/australia.yml', record: :once) do
+      TradeLead::AustraliaData.new(
+        "#{Rails.root}/spec/fixtures/trade_leads/australia/trade_leads.csv").import
+    end
```

Comment:
You might find it cleaner in the long-term to use this: https://www.relishapp.com/vcr/vcr/docs/test-frameworks/usage-with-rspec-metadata